eng/tools/lint-diff/src/markdown-utils.ts

```diff
@@ -85,7 +84,10 @@ export async function getRelatedArmRpcFromDoc(ruleName: string): Promise<string[
   const rpcRules: string[] = [];
   let response;
   try {
-    response = await axios.get(docUrl);
+    response = await fetch(docUrl);
+    if (!response.ok) {
+      throw new Error(`HTTP ${response.status}: ${response.statusText}`);
```

Comment:
It may look unusual to throw here, and catch a few lines later, but this is exactly what's recommended in the official docs for the API:

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch